packages/frameworks/src/frameworks.ts

```diff
@@ -1992,7 +1992,7 @@ export const frameworks = [
     logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/vite.svg',
     tagline:
       'Vite is a new breed of frontend build tool that significantly improves the frontend development experience.',
-    description: 'A Vue.js app, created with Vite.',
+    description: 'A Vue.js or React app, created with Vite.',
```

Comment:
The description here is for the `demo: https://vite-vue-template.vercel.app` which comes from the examples repo. Since the example is using Vue.js, the description should remain

https://vercel.com/new/templates

That being said, do we need two different detection mechanisms for vue and react. Or do they both share the same settings/routes?

Reply:
They definitely share the same settings and routes.